src/components/CreateEnvironment/EnvironmentSettings/index.tsx

```diff
@@ -30,15 +30,15 @@ function EnvironmentSettings(props: EnvironmentSettingsProps) {
   const { username, groups } = useContext(UserContext);
 
   useEffect(() => {
-    props.setPath("");
+    props.setPath(props.path);
   }, [username]);
 
   return (
     <Grid container spacing={1}>
       <Grid item xs={12} sm={4} md={3} textAlign={{ sm: "right" }}>
         <Box pr={3} pb={3} display="flex" justifyContent="flex-end">
           Name:
-          <HelpIcon title="Choose a name for your environment" />
+          <HelpIcon title="Choose a name for your environment, excluding -version" />
```

Comment:
Is the dash before version intended?

Reply:
Yes, it is. We want to ask users to remove a version as well as a dash

Reply:
Gotcha.